File: asakatsu/20221006/3.rs
// https://atcoder.jp/contests/tenka1-2012-qualA/tasks/tenka1_2012_qualA_2

// solve
fn main() {
    let mut s = String::new();
    std::io::stdin().read_line(&mut s).ok();

    let ans: Vec<&str> = s.split_whitespace().collect();
    println!("{}", ans.join(","));
}

File: asakatsu/20221127/3.py
# https://atcoder.jp/contests/arc011/tasks/arc011_1

m, n, N = map(int, input().split())

ans = N
while N >= m:
    make = (N // m) * n
    ans += make
    N -= (N // m) * m
    N += make

print(ans)
